src/components/ont/runs/ONTFlowcell.vue

```diff
@@ -131,80 +133,81 @@ export default {
     },
     flowcellId: {
       get() {
-        const flowcell = this.currentRun().flowcell_attributes.find(
-          (flowcell) => flowcell.position == this.position,
-        )
-        if (flowcell) {
-          return flowcell.flowcell_id
+        if (this.currentRun) {
```

Comment:
Not entirely clear what has changed here. 

A way to remove all of the logic would be to create a null object for current run which would return everything you need when currentRun is undefined.